### ISSUE TITLE
Make non-coprime error more verbose

### ISSUE THREAD

**Issue by metalhead777** [inIT-HF/gr-corrsounder#8]
As implemented in #7, trying to generate an FZC sequence with two non-coprime parameters ```sequence_length``` and ```q``` will return ```-1``` to indicate an error.

The function ```def sequence_frank_zadoff_chu``` in [corrsounder.py](/inIT-HF/gr-corrsounder/blob/master/python/pycorrsounder/corrsounder.py) needs to generate a verbose error message telling the user that they tried to use two non-coprime numbers.

**Comment by metalhead777** [comment]
Possible built-in exception to be used: 
```python
exception ValueError

raise ValueError('The sequence length {!s} and the parameter {!q} must be coprime!'.format(sequence_length, q))
```

**Comment by metalhead777** [comment]
Currently, error will only be displayed when using QT GUI or WX GUI: 
```bash
Traceback (most recent call last):
(...)
File "/home/user/prefix/lib/python2.7/dist-packages/pycorrsounder/corrsounder.py", line 16, in sequence_frank_zadoff_chu
    raise ValueError('The sequence length {!s} and the parameter {!q} must be coprime!'.format(sequence_length, q))
ValueError: Unknown conversion specifier q

>>> Done
```

**Comment by metalhead777** [comment]
Fixed with commit b776fdf591e793c01281b46429d1479435df14b4
